templates/todo/api/csharp/Program.cs

```diff
@@ -1,36 +1,38 @@
-using Azure.Identity;
-using Microsoft.ApplicationInsights.AspNetCore.Extensions;
-using MongoDB.Driver;
-using SimpleTodo.Api;
-
-var builder = WebApplication.CreateBuilder(args);
-var credential = new DefaultAzureCredential();
-builder.Configuration.AddAzureKeyVault(new Uri(builder.Configuration["AZURE_KEY_VAULT_ENDPOINT"]), credential);
-
-builder.Services.AddSingleton<ListsRepository>();
-builder.Services.AddSingleton(_ => new MongoClient(builder.Configuration[builder.Configuration["AZURE_COSMOS_CONNECTION_STRING_KEY"]]));
-builder.Services.AddControllers();
-builder.Services.AddApplicationInsightsTelemetry(builder.Configuration);
-
-var app = builder.Build();
-
-app.UseCors(policy =>
-{
-    policy.AllowAnyOrigin();
-    policy.AllowAnyHeader();
-    policy.AllowAnyMethod();
-});
-
-// Swagger UI
-app.UseSwaggerUI(options => {
-    options.SwaggerEndpoint("./openapi.yaml", "v1");
-    options.RoutePrefix = "";
-});
-
-app.UseStaticFiles(new StaticFileOptions{
-    // Serve openapi.yaml file
-    ServeUnknownFileTypes = true,
-});
-
-app.MapControllers();
+using Azure.Identity;
+using Microsoft.Azure.Cosmos;
+using SimpleTodo.Api;
+
+var builder = WebApplication.CreateBuilder(args);
+var credential = new DefaultAzureCredential();
+builder.Configuration.AddAzureKeyVault(new Uri(builder.Configuration["AZURE_KEY_VAULT_ENDPOINT"]), credential);
+
+builder.Services.AddSingleton<ListsRepository>();
+builder.Services.AddSingleton(_ => new CosmosClient(builder.Configuration[builder.Configuration["AZURE_COSMOS_CONNECTION_STRING_KEY"]]));

```

Comment:
Reason for switching from Mongo to Cosmos client?  @karolz-ms might know why we went with MongoClient  initially.

Reply:
Removing this template in another PR per @jongio